pkg/lint/lint.go

```diff
@@ -0,0 +1,180 @@
+package lint
```

Comment:
Functions that do the linting go here. They are not overly complex, but require some understanding of my chosen nomenclature. Don't worry, I plan on improving the names so it's clear what's what.